packaging/tests/tas-webdav/src/test/java/org/alfresco/webdav/WebDavTest.java

```diff
@@ -36,4 +44,16 @@ public void setup() throws Exception
         // The webdav protocol is enabled by default.
         //webDavProtocol.assertThat().protocolIsEnabled();
     }
-}
+
+    @BeforeMethod(alwaysRun=true)
+    public void showStartTestInfo(Method method)
+    {
+        LOG.info(String.format("*** STARTING Test: [%s] ***",method.getName()));
```

Comment:
Let's add a whitespace between the first and second argument of the method.

Reply:
Done.